### ISSUE TITLE
Schema must contain uniquely named types but contains multiple types named "ImageFormat"

### ISSUE THREAD

**Issue by damnitrahul** [reactgraphqlacademy/gatsby-source-sanity-transform-images#9]
Facing error issue after adding `gatsby-source-sanity-transform-images` plugin. 

````
Error 
UNHANDLED REJECTION Schema must contain uniquely named types but contains multiple types named "ImageFormat".



  Error: Schema must contain uniquely named types but contains multiple types named "ImageFormat".

  - Array.reduce

  - Array.reduce

  - SchemaComposer.js:130 SchemaComposer.buildSchema
    [Gatsby Blog]/[graphql-compose]/lib/SchemaComposer.js:130:12

  - schema.js:807 addCustomResolveFunctions
    [Gatsby Blog]/[gatsby]/dist/schema/schema.js:807:45

  - schema.js:227 updateSchemaComposer
    [Gatsby Blog]/[gatsby]/dist/schema/schema.js:227:9

  - schema.js:95 async buildSchema
    [Gatsby Blog]/[gatsby]/dist/schema/schema.js:95:3

  - index.js:138 async Object.build
    [Gatsby Blog]/[gatsby]/dist/schema/index.js:138:18

  - index.js:419 async module.exports
    [Gatsby Blog]/[gatsby]/dist/bootstrap/index.js:419:3

  - develop.js:441 async module.exports
    [Gatsby Blog]/[gatsby]/dist/commands/develop.js:441:7
````

Plugins List 

````
plugins: [
    `gatsby-plugin-react-helmet`,
    {
      resolve: `gatsby-source-filesystem`,
      options: {
        name: `images`,
        path: `${__dirname}/src/images`
      }
    },
    `gatsby-transformer-sharp`,
    `gatsby-plugin-sharp`,
    {
      resolve: `gatsby-plugin-manifest`,
      options: {
        name: `gatsby-starter-default`,
        short_name: `starter`,
        start_url: `/`,
        background_color: `#663399`,
        theme_color: `#663399`,
        display: `minimal-ui`,
        icon: `src/images/gatsby-icon.png` // This path is relative to the root of the site.
      }
    },
    {
      resolve: 'gatsby-source-sanity',
      options: {
        projectId: 'rhqgrccr',
        dataset: 'production',
        token: process.env.MY_SANITY_TOKEN
      }
    },
    'gatsby-source-sanity-transform-images'
        }
      }
    }
  ]
````

**Comment by maru3l** [comment]
This issue have been solved with the PR #8

**Comment by alexlobera** [comment]
Thanks @maru3l for answering this. I'll close it since it's already fixed
